flink-connectors/flink-connector-hbase/src/main/java/org/apache/flink/connector/hbase/source/reader/HBaseSourceEvent.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connector.hbase.source.reader;
+
+import org.apache.flink.connector.hbase.HBaseEvent;
+
+import org.apache.hadoop.hbase.Cell;
+
+import java.util.Arrays;
+
+/** The HBaseSourceEvent is used to represent incoming events from HBase. */
+public class HBaseSourceEvent extends HBaseEvent {
+
+    private final String table;
+    private final long timestamp;
+    /** Index of operation inside one wal entry. */
+    private final int index;
+
+    public HBaseSourceEvent(
+            Cell.Type type,
+            String rowId,
+            String table,
+            String cf,
```

Comment:
cf -> column family

Reply:
✅ in 5ba3ba8f10afffc296440e31b51eda0bc411ec22